pkg/channel/message_receiver.go

```diff
@@ -155,12 +154,10 @@ func (r *MessageReceiver) ServeHTTP(response nethttp.ResponseWriter, request *ne
 	}
 
 	transformers := append(defaultTransformers, AddHistory(host)...)
-	span := trace.FromContext(request.Context())
```

Comment:
I think this is suppose to pull the active span from the request context?

Reply:
I'm not sure, do i need to manually create the server span or not?